src/main.rs

```diff
@@ -865,6 +875,26 @@ async fn run_with_tracing() -> Result<(), Error> {
     run().await
 }
 
+async fn report_eip_quota_status(
+    ec2_client: &Ec2Client,
+    quota_client: &ServiceQuotaClient,
+) -> Result<(), Error> {
+    let addresses_result = ec2_client.describe_addresses().send().await?;
+    let allocated = addresses_result.addresses().unwrap_or_default().len();
+    let quota_result = quota_client
+        .get_service_quota()
+        .service_code("ec2")
+        .quota_code("L-0263D0A3")
```

Comment:
Well this is just classic AWS. I assume there isn't a constant for this in the AWS SDK?